### PR TITLE
More AFNI examples and update of functions

### PR DESCRIPTION
**Pull request by hermann74**
- Add more afni narps examples, afni function descriptions are added : 
       - 3dMatch
       - 3dpc
       -  2dcat
       -  IsoSurface
       -  suma
- Clean more useless command (wihtout inputs and outputs) : 
       -  cat <<EOF  .... EOF
       -  printf
       - \rm
       -  drop multiple space between args

### CODE REVIEW

**Comment by cyril-data** [comment]
Good work for me 👍.

File: launch_parser_on_nidm.py [hunk 1]
```diff
@@ -75,7 +76,7 @@ def main():
                 fsl_to_bids_prov(root + "/" + str(file), CONTEXT_URL, output_file=output_jsonld, verbose=opt.verbose)
                 output_png = output_dir_fsl + "/" + filename_ss_ext + ".png"
 
-            if file.endswith("proc.sub_001"):
+            elif file.endswith("proc.sub_001") or file.endswith(".tcsh"):
```
Comment (cyril-data) [inline launch_parser_on_nidm.py:79]: ok for the new extension ".tcsh"

**Review by cyril-data — APPROVED** [review]
Ok for me for these code factorization and add of documentation.

**Comment by cmaumet** [comment]
Excellent, let's merge!

**Pull request merged**